config/config-controller.yaml

```diff
@@ -25,7 +25,7 @@ data:
   queueSidecarImage: github.com/knative/serving/cmd/queue
 
   # Comment the following lines to disable the single-tenant autoscaler.
```

Comment:
Update this comment to say "uncomment .. to enable ..."?

Reply:
Done.